powerscale/helper/nfs_global_settings.go

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
the file name better to keep consistent with others

Reply:
Modified. Thanks!